cmd/commandfuncs.go

```diff
@@ -506,6 +506,15 @@ func cmdValidateConfig(fl Flags) (int, error) {
 	validateCmdConfigFlag := fl.String("config")
 	validateCmdAdapterFlag := fl.String("adapter")
 
+	if validateCmdConfigFlag == "" {
+		validateCmdConfigFlag = "Caddyfile"
+	}
```

Comment:
`loadConfig()` already does this.

Reply:
@mholt doing it because on the next line we do the `os.Stat` validation. Didn't think we want to run an `os.Stat` for a empty string. wdty?

Reply:
We shouldn't be duplicating the logic that is in `loadConfig()`.

Reply:
@mholt Not sure if got it. Do prefer to avoid this change at all? And you okay with the current return from `loadConfig`? 

Or you okay with the change, but doesn't think we should set the `validateCmdConfigFlag` here?

Thanks

Reply:
The `loadConfig()` function already accounts for whether a config file was specified, or if a config file is not found (as you noted in your OP). This code steps on that logic, since now there are two places where we assume a default config file instead of one.

The change should be made in the one place that an explicitly-defined config file is loaded and, if it is not found, then return an error.

Reply:
@mholt gotcha! let me see if I can fit this in a better way.